.github/ISSUE_TEMPLATE/03-documentation.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: Documentation request
+description: A request for new or updated documentation.
+labels: A-docs
+body:
+  - type: textarea
+    attributes:
+      label: What pages are affected?
+  - type: textarea
+    attributes:
+      label: What needs to be changed or added?
+  - type: textarea
+    attributes:
+      label: Feature description
+  - type: input
+    attributes:
+      label: Engineering tracking ticket
+  - type: textarea
+    attributes:
+      label: Design documents
+      description: |
+        Please link any available design documents, like technical specs, product specs, or test files.
```

Comment:
```suggestion
  - type: textarea
    attributes:
      label: What pages are affected?
  - type: textarea
    attributes:
      label: What needs to be changed or added?
  - type: input
    attributes:
      label: Engineering tracking ticket
   - type: textarea
    attributes:
      label: Feature description
  - type: textarea
    attributes:
      label: Design documents
      description: |
        Please link any available design documents, like technical specs, product specs, or test files.
```

Reply:
Minor, but it's more intuitive to me to have the issue first and everything else afterwards.

Reply:
Sounds good to me! I figured you might want to do some deeper surgery on these forms, and particular this one, so I just stuck with a pretty rote translation of the existing templates.